packages/caip/src/adapters/mtpelerin/index.ts

```diff
@@ -0,0 +1,23 @@
+import { AssetId } from './../../assetId/assetId'
+
+import {
+    avalancheAssetId,
+    btcAssetId,
+    ethAssetId,
+  } from '../../constants'
+
+
+const AssetIdToMtPelerinTickerMap = {
+    [avalancheAssetId]: 'AVAX',
+    [btcAssetId]: 'BTC',
+    [ethAssetId]: 'ETH',
+} as Record<AssetId, string>
```

Comment:
Can we update the PR description with a reference to the MtPelerin docs where these are defined?